src/api/l_data_soundData.c

```diff
@@ -46,9 +47,11 @@ static int l_lovrSoundDataSetSample(lua_State* L) {
   return 0;
 }
 
-static int l_lovrSoundDataGetPointer(lua_State* L) {
+static int l_lovrSoundDataGetBlob(lua_State* L) {
   SoundData* soundData = luax_checktype(L, 1, SoundData);
-  lua_pushlightuserdata(L, soundData->blob.data);
+  Blob* blob = soundData->blob;
+  luax_pushtype(L, Blob, blob);
+  lovrRelease(Blob, blob);
```

Comment:
I don't think releasing the Blob is wanted here.  If `getBlob` was called multiple times, eventually it would get freed.  Usually `lovrRelease` is used in the API if we've just created a new object and have no need for C to keep its reference around.

Reply:
Ah I just thought I saw it used like that somewhere else; figured maybe luax_pushtype needed to be balanced (that's one of the pieces of magic I haven't dug too deep into yet). Okay, I'll just remove it from sounddata and texturedata!